international-brazilian-journal-of-urology.csl

```diff
@@ -0,0 +1,172 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" page-range-format="minimal-two" demote-non-dropping-particle="never" default-locale="en-US" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>International Brazilian Journal of Urology</title>
```

Comment:
@adam3smith, @damnation333, any thoughts on the title? It looks like the full title is really just "International Braz J Urol". Shall we go with "International Brazilian Journal of Urology" regardless? 

It's rather uncommon in that the journal doesn't seem to use the unabbreviated title anywhere. See also
https://www.ncbi.nlm.nih.gov/nlmcatalog/?term=1677-5538 and http://www.intbrazjurol.com.br/.